mesheryctl/cmd/perf.go

```diff
@@ -79,17 +84,26 @@ var perfCmd = &cobra.Command{
 		if len(testURL) > 0 {
 			postData = postData + "\nendpoint_url: " + testURL
 		} else {
-			println("Error: Please enter a TestURL")
+			println("\nError: Please enter a test URL")
 			return
 		}
 
+		// Methord to check if the entered Test URL is valid or not
+		var validURL bool = govalidator.IsURL(testURL)
+
+		if (validURL == false) {
+			println("\nError: Please enter a valid test URL")
+			return
+		}
+
+
 		postData = postData + "\nclient:"
 		postData = postData + "\n connections: " + concurrentRequests
 		postData = postData + "\n rps: " + qps
 
 		req, err := http.NewRequest("POST", mesheryURL, bytes.NewBuffer([]byte(postData)))
 		if err != nil {
-			println("Error in building the request")
+			println("\nError in building the request")
```

Comment:
Do you know under what circumstances this might fail? Are we able to pass along additional error info to the user?

Reply:
It would be great if err was also printed. In that case we won't have to handle different scenarios.

Reply:
We can print the error in next line under 'error message'